File: scratch/get_lj_db_creds.php
<?php
require_once dirname(__DIR__) . '/config.php';

try {
    $db = new PDO("mysql:host=" . DB_HOST . ";dbname=" . DB_NAME . ";charset=utf8", DB_USER, DB_PASS);
    $db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
    
    // Find livejournal accounts
    $stmt = $db->prepare("SELECT * FROM platform_accounts WHERE platform = 'livejournal'");
    $stmt->execute();
    $rows = $stmt->fetchAll(PDO::FETCH_ASSOC);
    foreach ($rows as $row) {
        echo "Username: " . $row['username'] . " | Password: " . $row['password'] . "\n";
    }
} catch (Exception $e) {
    echo "Error: " . $e->getMessage() . "\n";
}

File: scratch/get_lj_db_creds.py
import pymysql
import json

try:
    conn = pymysql.connect(
        host="127.0.0.1",
        port=3307,
        user="root",
        password="",
        database="seo_system",
        cursorclass=pymysql.cursors.DictCursor
    )
    with conn.cursor() as cursor:
        cursor.execute("SELECT username, password FROM social_accounts WHERE platform = 'livejournal'")
        accounts = cursor.fetchall()
        import base64
        for acc in accounts:
            try:
                pw_decoded = base64.b64decode(acc['password']).decode('utf-8')
            except Exception:
                pw_decoded = acc['password']
            print(f"Username: {acc['username']}, Password: {pw_decoded}")
except Exception as e:
    print(f"Database error: {e}")
